### DISCUSSION TITLE
Appears the latest version isn't limiting the file download quantity that was set.

### DISCUSSION THREAD

**Discussion by 98gmarquee** [aclap-dev/video-downloadhelper#102] [Help and Q&A]
### Your question

I have mine set to 4 files maximum, then it would queue the rest if there were more than 4. It's currently downloading many more.

**Comment by paulrouget** [comment]
> Addon & CoApp version and details
> No response

I don't know which browser you're using, on what OS, and what version.

Please share info: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

**Reply by 98gmarquee** [reply]
Firefox 122.0.1

Edition	Windows 10 Home
Version	22H2
Installed on	‎6/‎11/‎2020
OS build	19045.4046
Experience	Windows Feature Experience Pack 1000.19053.1000.0

I reinstalled the old version (8.1.4.0) and it's working fine again.

**Comment by paulrouget** [comment]
As of now, with version 8.2, it doesn't queue for you?

What's the value you set for "Max concurrent downloads" (make sure you set that pref, not "Max concurrent stream downloads")?

Also, share info as explained here: https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F (please do it, it will help a lot to figure out your issue)

**Reply by 98gmarquee** [reply]
I had both set to 4. Max concurrent download and max concurrent stream downloads.

This is the version I changed back to that does limit.

```
version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Bryan",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

```
